samples: bluetooth: nrf_dm: Add Zero-distance calibration note (#9549)
nrfconnect/sdk-nrf: samples/bluetooth/nrf_dm/README.rst

When a device does not respond within a certain period of time, it is removed from the list.
Also the last measurement results with the peers are stored in this list.

Zero-distance calibration
*************************

The measured values of distance measurement include an offset.
The offset value may depend on the design of the radio circuit, the antenna used, or the PCB layout.
When using a library and a sample, you need to compensate the zero-distance offset and perform the calibration manually.
To determine the offset value, place the devices at a reference distance of 60 cm from each other and collect a batch of measurement data.

> The difference between the averaged measurement value and the actual distance is the offset.

Reviewer: I would say "The difference between the averaged measurement value and the actual distance can be used as an estimate of the offset."

Author: ACK

> The difference between the averaged measurement value and the actual distance can be used as an estimate of the offset.

Reviewer 2: ```suggestion
The difference between the average measurement value and the actual distance can be used as an estimate of the offset.
```

Author: Fixed.

When a device does not respond within a certain period of time, it is removed from the list.
Also the last measurement results with the peers are stored in this list.

Zero-distance calibration
*************************

> The measured values of distance measurement include an off-set.

Reviewer 3: I think it is usually written "offset" without the hyphen

Author: Fixed.

> The offset value may be due to the design of the radio circuit, the antenna used or the designed PCB Layout.

Reviewer 2: ```suggestion
The offset value may depend on the design of the radio circuit, the antenna used, or the PCB layout.
```

> Zero-distance value should be handled outside the library and sample.

Reviewer 3: ```suggestion
Zero-distance offset should be compensated for by the user of the library and sample.
```

> Zero-distance offset should be compensated for by the user of the library and sample.

Reviewer 2: ```suggestion
When using a library and a sample, you need to compensate the zero-distance offset and perform the calibration manually.
```

> You should manually perform the calibration.

Reviewer 3: ```suggestion
Users of the library need to manually perform the calibration.
```

> Users of the library need to manually perform the calibration.

Reviewer 2: ```suggestion
```
Okay to combine this with what was said below, as I'm suggesting? Does it change the meaning?

Author: I think this sentence is not needed.

> To determine the off-set value, place the devices at reference distance and collect a batch of measurement data.

Reviewer 3: Should we mention the suggested distance here, I think Carsten mentioned 60 cm?

Author: Good point. Added a note about the suggested distance.

> To determine the offset value, place the devices at reference distance, the suggested distance is 60cm and collect a batch of measurement data.

Reviewer 2: ```suggestion
To determine the offset value, place the devices at a reference distance of 60 cm from each other and collect a batch of measurement data.
```
